src/server/api/controllers/questions.controller.js

```diff
@@ -0,0 +1,14 @@
+const knex = require('../../config/db');
```

Comment:
@dpfernandes It looks like we are all adding this line into our PR's. Which issue did you think covered the addition of this line?